plugins/modules/notification/cisco_webex.py

```diff
@@ -0,0 +1,189 @@
+---
+module: cisco_webex
+short_description: Send a message to a Cisco Webex Teams Room or Individual
+description:
+    - Send a message to a Cisco Webex Teams Room or Individual with options to control the formatting.
+author: Drew Rusell (@drew-russell)
+notes:
+  - The C(recipient_id) type must be valid for the supplied C(recipient_id).
+  - Full API documentation can be found at U(https://developer.webex.com/docs/api/basics).
+
+options:
+
+  recipient_type:
+    description:
+       - The request parameter you would like to send the message to.
+       - Messages can be sent to either a room or individual (by ID or E-Mail).
+    required: yes
+    choices: ['roomId', 'toPersonEmail', 'toPersonId']
+    type: str
+
+  recipient_id:
+    description:
+      - The unique identifier associated with the supplied C(recipient_type).
+    required: yes
+    type: str
+
+  msg_type:
+    description:
+       - Specifies how you would like the message formatted.
+    default: text
+    choices: ['text', 'markdown']
+    type: str
+    aliases: ['message_type']
+
+  personal_token:
+    description:
+      - Your personal access token required to validate the Webex Teams API.
+    required: yes
+    aliases: ['token']
+    type: str
+
+  msg:
+    description:
+      - The message you would like to send.
+    required: yes
+    type: str
+    aliases: ['message']
+'''
+
+EXAMPLES = """
+# Note: The following examples assume a variable file has been imported
+# that contains the appropriate information.
+
+- name: Cisco Webex Teams - Markdown Message to a Room
+  community.general.cisco_webex:
+    recipient_type: roomId
+    recipient_id: "{{ room_id }}"
+    msg_type: markdown
+    personal_token: "{{ token }}"
+    msg: "**Cisco Webex Teams Ansible Module - Room Message in Markdown**"
+
+- name: Cisco Webex Teams - Text Message to a Room
+  community.general.cisco_webex:
+    recipient_type: roomId
+    recipient_id: "{{ room_id }}"
+    msg_type: text
+    personal_token: "{{ token }}"
+    msg: "Cisco Webex Teams Ansible Module - Room Message in Text"
+
+- name: Cisco Webex Teams - Text Message by an Individuals ID
+  community.general.cisco_webex:
+    recipient_type: toPersonId
+    recipient_id: "{{ person_id}}"
+    msg_type: text
+    personal_token: "{{ token }}"
+    msg: "Cisco Webex Teams Ansible Module - Text Message to Individual by ID"
+
+- name: Cisco Webex Teams - Text Message by an Individuals E-Mail Address
+  community.general.cisco_webex:
+    recipient_type: toPersonEmail
+    recipient_id: "{{ person_email }}"
+    msg_type: text
+    personal_token: "{{ token }}"
+    msg: "Cisco Webex Teams Ansible Module - Text Message to Individual by E-Mail"
+
+"""
+
+RETURN = """
+status_code:
+  description:
+    - The Response Code returned by the Webex Teams API.
+    - Full Response Code explanations can be found at U(https://developer.webex.com/docs/api/basics).
+  returned: always
+  type: int
+  sample: 200
+
+msg:
```

Comment:
This change is still missing:
```suggestion
message:
```